var/spack/repos/builtin/packages/py-nbval/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyNbval(PythonPackage):
+    """Py.test plugin for validating Jupyter notebooks.
+
+    The plugin adds functionality to py.test to recognise and collect Jupyter notebooks.
+    The intended purpose of the tests is to determine whether execution of the stored
+    inputs match the stored outputs of the .ipynb file. Whilst also ensuring that the
+    notebooks are running without errors.
+    """
+
+    homepage = "https://github.com/computationalmodelling/nbval"
+    pypi     = "nbval/nbval-0.9.6.tar.gz"
+
+    version('0.9.6', sha256='cfefcd2ef66ee2d337d0b252c6bcec4023384eb32e8b9e5fcc3ac80ab8cd7d40')
+
+    depends_on('python@3.6:3.99', type=('build', 'run'))
```

Comment:
Seems like they still support 2, I would remove this line

Reply:
Again, I got that from the master brnach. There seems to be some consistency to my errors.